README.md

```diff
@@ -252,3 +257,36 @@ async fn main() {
   ```bash
   rs-wnfs coverage
   ```
+
+## Contributing
+
+### Pre-commit Hook
+
+This library recommends using [pre-commit](https://pre-commit.com/) for running pre-commit hooks. Please run this before every commit and/or push.
+
+- Once installed, Run `pre-commit install` to setup the pre-commit hooks locally.  This will reduce failed CI builds.
+- If you are doing interim commits locally, and for some reason if you _don't_ want pre-commit hooks to fire, you can run
+  `git commit -a -m "Your message here" --no-verify`.
+
+## Getting Help
+
+For usage questions, usecases, or issues reach out to us in our [Discord webnative-fs channel](https://discord.com/channels/478735028319158273/877623827331428403).
+We would be happy to try to answer your question or try opening a new issue on Github.
+
+## External Resources
+
+This is a list of links to blog posts, confereence talks, and tutorials related to WNFS in general:
+
+### Talks
+
+- [A Distributed File System for Secure P2P Applications](https://www.youtube.com/watch?v=-f4cH_HQU4U) by Brooklyn Zelenka (Strange Loop 2022)
```

Comment:
There's also a short overview of WNFS from IPFS Thing: https://www.youtube.com/watch?v=3se17NAS-Lw

Reply:
good call, will add.